src/EnergyPlus/GroundTemperatureModeling/FiniteDifferenceGroundTemperatureModel.cc

```diff
@@ -832,7 +832,7 @@ void FiniteDiffGroundTempsModel::initDomain(EnergyPlusData &state)
     // SUBROUTINE LOCAL VARIABLE DECLARATIONS:
 
     // Temporary KA model for initialization
-    auto tempModel = std::make_unique<KusudaGroundTempsModel>();
+    auto tempModel = std::make_unique<KusudaGroundTempsModel>(); // (AUTO_OK) Why does this have to be a unique_ptr?
```

Comment:
I am using `grep` to these "bad" `auto` idioms using. If I can't fix the `auto` for some reason, I leave this comment so that my `grep` doesn't find it again but so that I can go back later and fix it if I think of a way. In this case, I am also questioning why this needs to be a `unique_ptr`. A question for another day.